src/twitchLanguageServer.ts

```diff
@@ -78,10 +78,25 @@ function onTtvChatJoin(channel: string, username: string, self: boolean) {
 
 function onTtvChatMessage(channel: string, user: any, message: string) {
   const userName = user['display-name'] || user.username;
-  parseMessage(userName, message);
+  parseMessage(channel, userName, message);
 }
 
-export function parseMessage(userName: string, message: string) {
+export function parseMessage(channel:string, userName: string, message: string) {
+
+  /**
+   * Regex pattern to verify the command is a highlight command without
+   * any arguments. This will send a 'howto' message back on chat to
+   * inform the users how to use the highlight command.
+   *
+   * Matches:
+   *
+   * !line
+   */
+  const commandOnlyPattern = /^!(?:line|highlight)$/;
```

Comment:
Maybe we should add the `i` flag to the patterns to allow case insensitive checking? So it doesn't matter if someone writes `!liNe` or `!line`

Reply:
Thanks @cedrickring, I've made the change in the lasted commit!